cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/Handler.kt

```diff
@@ -166,6 +167,13 @@ abstract class Handler<ResultNode : Node?, HandlerNode, L : LanguageFrontend<in
         return null
     }
 
+    /** Creates a random name starting with a prefix plus a random UUID (version 4). */
+    fun getRandomTempName(): Name {
+        val prefix = "TEMP-RANDOM-NAME"
```

Comment:
Any chance to make this constant available for tests as well? I tried moving it to the class, but I obviously do not have a handler available during the tests. Maybe there is a better place for this?